distribution/server/src/assemble/LICENSE.bin.txt

```diff
@@ -446,7 +446,7 @@ The Apache Software License, Version 2.0
  * SnakeYaml -- org.yaml-snakeyaml-1.27.jar
  * RocksDB - org.rocksdb-rocksdbjni-6.10.2.jar
  * Google Error Prone Annotations - com.google.errorprone-error_prone_annotations-2.5.1.jar
- * Apache Thrifth - org.apache.thrift-libthrift-0.12.0.jar
+ * Apache Thrifth - org.apache.thrift-libthrift-0.14.2.jar
```

Comment:
Apache Thrift?

Reply:
it is a dependency of DistributedLog, used by the StreamStorage Service, used by Pulsar Functions